server/app/controllers/FeatureFlagOverrideController.java

```diff
@@ -0,0 +1,32 @@
+package controllers;
+
+import annotations.FeatureFlagOverrides;
+import auth.Authorizers;
+import javax.inject.Inject;
+import org.pac4j.play.java.Secure;
+import play.mvc.Http.Request;
+import play.mvc.Result;
+
+/**
+ * Allows for overriding of feature flags by the CiviForm Admin via HTTP request.
+ */
+public class FeatureFlagOverrideController extends CiviFormController {
+  private final FeatureFlagOverrides overrides;
+
+  @Inject
+  public FeatureFlagOverrideController(FeatureFlagOverrides overrides) {
+    this.overrides = overrides;
+  }
+
+  @Secure(authorizers = Authorizers.Labels.CIVIFORM_ADMIN)
+  public Result enable(Request request, String flagName) {
```

Comment:
Here and in the disable handler, I'd extend `DevController` and call `isDevOrStagingEnvironment()` for consistency.

Reply:
done!